staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go

```diff
@@ -229,6 +229,9 @@ func (o *ScaleOptions) RunScale() error {
 	}
 
 	if len(infos) == 0 {
+		if infoErr != nil {
+			return fmt.Errorf("no objects passed to scale %w", infoErr)
```

Comment:
I'd probably reword this:

    failed to resolve resources: %w

Because I did pass objects; it's just there was an error failing to query them.